### DISCUSSION TITLE
API for the missal

### DISCUSSION THREAD

**Discussion by formijin** [DivinumOfficium/divinum-officium#2587] [General]
Helo please is there an API that we can use to access the missal and embed its daily readings to a website

**Comment by gregordick** [comment]
Hello, and thanks for your interest.  I'm afraid, though, that no API is exposed at present.

**Reply by ajmalton** [reply]
It is a goal of our work, to expose an API to deliver the content by liturgical action and date and version and language, in a useable semantic-HTML format -- eventually.

In the meantime you can retrieve the Mass using GET (eg from curl or requests.get) and a URL like this:

https://divinumofficium.com/cgi-bin/missa/missa.pl?date=12-25-2020&version=1960&lang2=Latin

and then scrape out what you need.  A bit ad hoc, but not too difficult.

**Comment by formijin** [comment]
Thank you very much for the help.
Please keep us update on any progress made
Nneji Fortunate Jideofor

**Comment by formijin** [comment]
Thanks very much for the help. Please keep us update on the progress.. thanks a million
